src/modules/Brightness/Module.js

```diff
@@ -1,15 +1,18 @@
 /*
 * Changes the Image Brightness
 */
-
+const info = require('./info.json');
```

Comment:
Second idea - could we combine this with the `const defaults` line to have an absolute minimum of additions required in a module to access the defaults?

Reply:
I will make the changes! Thanks!